docs/user-guides/pallet-proxy.md

```diff
@@ -0,0 +1,191 @@
+
+5. **IdentityJudgement** - This proxy type allows a single functionality to delegate account which is to provide a judgement to an identity in a naming system. The delegator must be a registrar.
+
+6. **CancelProxy** - This proxy type allows delegate account to  to reject and remove any announced proxy calls.
+
+7. **DappsStaking** - This proxy type allows delegate account to perform DappStaking related transactions such as register, unregister contracts, claim rewards etc.
+
+
+**Extrinsics**
+---
+The proxy pallet provides the following extrinsics (functions):
+* **addProxy**(delegate, proxyType, delay) - allows the sender to register a proxy account capable of making calls on behalf of the sender. If the delay value is set to a number greater than 0, the proxy account must announce a transaction and wait for the specified number of blocks before executing it as a proxy. This function also emits a ProxyAdded event.
+
+* **announce**(real, callHash) - used to record the announcement of a proxy transaction made by proxy accounts that have a delay requirement. This function emits an Announced event.
+
+* **anonymous**(proxyType, delay, index) - generates a new account with an inaccessible private key, making it inaccessible. The sender assumes the role of a proxy for this account, based on the specified type and delay. Caution: If the proxy is removed, the primary account will become inaccessible. This function emits an AnonymousCreated event.
+
+* **killAnonymous**(spawner, proxyType, index, height, extIndex) - removes a previously spawned anonymous proxy.
+
+* **proxy**(real, forceProxyType, call) - performs a transaction as a proxy. This function emits a ProxyExecuted event.
+
+* **proxyAnnounced**(delegate, real, forceProxyType, call) - executes a transaction as a proxy and removes any previous corresponding announcements. This function emits a ProxyExecuted event.
+
+* **rejectAnnouncement**(delegate, callHash) - if the sender is a prime account, it removes a specific announcement from their proxy account. 
+
+* **removeAnnouncement**(real, callHash) - if the sender is a proxy account, it removes a specific announcement to their prime account.
+
+* **removeProxies()** - unregisters all proxy accounts associated with the sender.
+
+* **removeProxy**(delegate, proxyType, delay) - unregisters a specific proxy account linked to the sender. This function emits a ProxyRemoved event.
+
+
+**Constants for Pallet Proxy**
+---
+
+| Constant                 | Description                                                                                                  |
+|--------------------------|--------------------------------------------------------------------------------------------------------------|
+| announcementDepositBase  | Retrieves the base amount of currency required to reserve when creating an announcement.                     |
```

Comment:
Strange choice of words with _Retrieves_. :thinking:

Reply:
will choose more appropriate word. 
`transfer` should be fine imo.